### DISCUSSION TITLE
How to allow inbound and block outbound connection per application

### DISCUSSION THREAD

**Discussion by bitleader** [tnodir/fort#359] [Q&A] [answered]
Hello!

I would like to setup Fort to allow inbound connection only defined in a Zone and allow every outbound at the same time. Is this possible?

I noticed that if I define zone then connections is allowed on that zone for both direction but I want to filter only inbound connection not outbound.

Thanks.

**Comment by tnodir** [comment, marked as answer]
Hello and welcome!

- Create Allowed Preset Rule with specified Zones, named for example "Trusted Zone"
- Create Allowed App Rule with "dir(out)" text and ticked "Terminating Rule: Block", named for example "Allow Out OR Zone"
- Add the "Trusted Zone" preset rule into "Allow Out OR Zone" rule
- Set the "Allow Out OR Zone" rule to desired Program

So, the program can access Internet via any outbound connection OR only specified Zones.

**Reply by bitleader** [reply]
Cool 🤓 Thanks very much!
